src/main/java/core/basesyntax/service/operations/BalanceOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+//This class provides balance from transaction input file

```

Comment:
Comments in code is not a preferred way to describe what a class or a method does. It's better to use JavaDoc, it provides better readability and understanding.